lib/elixir/pages/Deprecations.md

```diff
@@ -0,0 +1,62 @@
+# Deprecations
+
+## Policy
+
+Elixir deprecations happens in 3 steps:
+
+  1. The feature is soft-deprecated. It means both CHANGELOG and documentation must list the feature as deprecated but no warning is effectively emitted by running the code. There is no requirement to soft-deprecate a feature.
+
+  2. The feature is effectively deprecated by emitting warnings on usage. In order to deprecate a feature, the proposed alternative MUST exist for AT LEAST two versions. For example, `Enum.uniq/2` was soft-deprecated in favor of `Enum.uniq_by/2` in Elixir v1.1. This means a deprecation warning may only be emitted by Elixir v1.3 or later.
+
+  3. The feature is removed. This can only happen on major releases. This means deprecated features in Elixir v1.x shall only be removed by Elixir v2.x.
+
+
+## Table of deprecations
+
+Deprecated feature        | Deprecated in | Replaced by             | Replacement available since
+:------------------------ | :------------ | :---------------------- | :---------------------------
+`Atom.to_char_list/1`     | [v1.5]        | `Atom.to_charlist/1`    | v1.3
+`Float.to_char_list/1`    | [v1.5]        | `Float.to_charlist/1`   | v1.3
+`GenEvent` module         | [v1.5]        | `Supervisor` and `GenServer`; [`GenStage`](https://hex.pm/packages/gen_stage); [`:gen_event`](http://www.erlang.org/doc/man/gen_event.html) | v1.0 (`Supervisor` and `GenServer`); v1.3 (`GenStage`); OTP 17 (`:gen_event`)
+`Integer.to_char_list/1`  | [v1.5]        | `Integer.to_charlist/1` | v1.3
+`Kernel.to_char_list/1`   | [v1.5]        | `Kernel.to_charlist/1`  | v1.3
+`String.to_char_list/1`   | [v1.5]        | `String.to_charlist/1`  | v1.3
+`()` to mean `nil`        | [v1.5]        | `nil`                   | v1.0
+`:as_char_lists` value in `t:Inspect.Opts.t/0` type | [v1.5] | `:as_charlists` | v1.3
+`:char_lists` key in `t:Inspect.Opts.t/0` type | [v1.5] | `:charlists` | v1.3
+`char_list/0` type        | [v1.5]        | `charlist/0` type       | v1.3
```

Comment:
since it's built in, it doesn't link to anything and it leaves the `t:`.
I think it's an ExDoc bug. While it doesn't link, it should still remove the `t:`